src/loader.js

```diff
@@ -16,13 +63,22 @@ export function loadFile(filepath) {
     throw new Error(`Unsupported filetype ${filepath}`);
 }
 
-export function loadLocale(locale, filepath) {
+export function loadLocale(locale, filepath, replaceVariablesNames = false) {
     if (typeof filepath === 'string') {
```

Comment:
I think it makes sense to move replaceVariablesNames logic to `addLocale` method.